cluster/os-3.9.0-alpha.4/provider.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+set -e
+
+prefix=kubevirt-os-3.9.0-alpha.4
+
+function _main_ip() {
+    echo 127.0.0.1
+}
+
+_cli='docker run --privileged --rm -v /var/run/docker.sock:/var/run/docker.sock rmohr/cli:latest'
+
+function up() {
+    # Add one, 0 here means no node at all, but in the kubevirt repo it means master-only
+    local num_nodes=${VAGRANT_NUM_NODES-0}
+    num_nodes=$((num_nodes + 1))
+    ${_cli} run --nodes ${num_nodes} --osp-port 127.0.0.1:8443 --ssh-port 127.0.0.1:2201 --background --registry-port 127.0.0.1:5000 --prefix $prefix --registry-volume kubevirt_registry --base "rmohr/os-3.9"
```

Comment:
can we get a sha256 tag on this os-3.9 container image

Reply:
I have a number of fixes under @rmohr repository when we will have them, we will use some sha tag.